tss-esapi/src/context/general_esys_tr.rs

```diff
@@ -85,6 +85,26 @@ impl Context {
         Ok(())
     }
 
+    #[cfg(has_esys_tr_get_tpm_handle)]
+    /// Retrieve the `TpmHandle` stored in the given object.
```

Comment:
We really need better documentation for these context methods. But it is not something that you need to change here. I have created an issue for that #364.